src/extension/options-page/DashboardInitSteps/Step1Ra.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import StepBase from './StepBase'
+import { Link, Redirect, useRouteMatch } from 'react-router-dom'
+import { DialogRouter } from '../DashboardDialogs/DialogBase'
+import { PersonaImportDialog } from '../DashboardDialogs/Persona'
+import ActionButton from '../DashboardComponents/ActionButton'
+import { geti18nString } from '../../../utils/i18n'
+
+const header = geti18nString('dashboard_advanced_restoration')
+
+const actions = (
+    <>
+        <ActionButton variant="outlined" color="default" component={Link} to="../1r">
+            {geti18nString('back')}
+        </ActionButton>
+        <span></span>
```

Comment:
Why <span /> is here?

Reply:
left button and 'right button'

Reply:
🤔 use it as a "flex: 1" to span the content?